frontend/menu.c

```diff
@@ -1569,6 +1574,7 @@ static menu_entry e_menu_adv_options[] =
 	//mee_onoff_h   ("SIO IRQ Always Enabled", 0, Config.Sio, 1, h_cfg_sio),
 	mee_onoff_h   ("SPU IRQ Always Enabled", 0, Config.SpuIrq, 1, h_cfg_spuirq),
 	//mee_onoff_h   ("Rootcounter hack",       0, Config.RCntFix, 1, h_cfg_rcnt1),
+	mee_onoff_h   ("No memory card fix",    0, Config.NoMemCard, 1, h_cfg_nomem),
 	mee_onoff_h   ("Rootcounter hack 2",     0, Config.VSyncWA, 1, h_cfg_rcnt2),
```

Comment:
The menu (extra stuff->memcard manager) already allows to set the memory card to 'none', do we really need another option?